scripts/build-scala-client.sh

```diff
@@ -5,53 +5,21 @@
 ROOT=$(pwd)
 SDIR=client/scala/armada-scala-client
 
-rm -rf $ROOT/$SDIR/proto
-mkdir -p $ROOT/$SDIR/proto
-
-cd proto
-for pfile in \
-    google/api/annotations.proto \
-    google/api/http.proto \
-    google/protobuf/*.proto \
-    github.com/gogo/protobuf/gogoproto/gogo.proto \
-    k8s.io/api/core/v1/generated.proto \
-    k8s.io/apimachinery/pkg/api/resource/generated.proto \
-    k8s.io/apimachinery/pkg/apis/meta/v1/generated.proto \
-    k8s.io/apimachinery/pkg/runtime/generated.proto \
-    k8s.io/apimachinery/pkg/runtime/schema/generated.proto \
-    k8s.io/apimachinery/pkg/util/intstr/generated.proto \
-    k8s.io/api/networking/v1/generated.proto
-do
-  dir=$(dirname $pfile)
-  mkdir -p $ROOT/$SDIR/proto/$dir
-  cp $pfile $ROOT/$SDIR/proto/$dir/
-done
-
-cd ..
-for pfile in \
-    pkg/api/event.proto pkg/api/submit.proto pkg/api/health.proto pkg/api/job.proto pkg/api/binoculars/binoculars.proto
-do
-  dir=$(dirname $pfile)
-  mkdir -p $ROOT/$SDIR/proto/$dir
-  cp $pfile $ROOT/$SDIR/proto/$dir/
-done
-
-# the sbt config in the $SDIR directory causes the following commands to generate
-# the scala protobuf files into $SDIR/target/scala-2.13/src_managed/main/
+# the mvn pom.xml in the $SDIR directory causes the following commands copy all relevant proto files
+# into $SDIR/src/main/protobuf and compile these protobuf files into Scala files into $SDIR/target/generated-sources/scala-2.13/
+# this then compils all Scala files into Java classes and packages everything into a jar file
 
 cd $ROOT/$SDIR
-sbt clean && \
-sbt -Dsbt.io.implicit.relative.glob.conversion=allow compile && \
-sbt -Dsbt.io.implicit.relative.glob.conversion=allow package
+mvn clean package
 
 if [ $? -eq 0 ]; then
-  jarfile=$(find . -type f -name 'armada-scala-client*.jar')
-  if [[ $jarfile ]]; then
+  jarfile=$(find . -type f -name 'armada-scala-client*.jar' | sort | head -n1)
```

Comment:
There are multiple jars (package, sources, javadocs), the first is the actual package.